Tribler/Test/Core/test_sqlitecachedb_upgrade.py

```diff
@@ -34,27 +43,24 @@ def tearDown(self):
         self.session.del_instance()
         self.session = None
 
-    def test_upgrade_from_obsolete_version(self):
-        """We no longer support DB versions older than 17 (Tribler 6.0)"""
-        db_path = init_bak_tribler_sdb(u"bak_old_tribler.sdb", destination_path=self.getStateDir(), overwrite=True)
-        db_script_path = os.path.join(self.session.get_install_dir(), DB_SCRIPT_RELATIVE_PATH)
-
+    def copy_and_initialize_upgrade_database(self, db_name):
+        shutil.copyfile(os.path.join(self.DATABASES_DIR, db_name),
+                        os.path.join(self.session_base_dir, 'tribler.sdb'))
+        db_path = os.path.join(self.session_base_dir, 'tribler.sdb')
+        db_script_path = os.path.join(self.session.get_install_dir())
         self.sqlitedb = SQLiteCacheDB(db_path, db_script_path)
         self.sqlitedb.initialize()
+        self.sqlitedb.initial_begin()
 
-        class MockTorrentStore(object):
-
-            def flush():
-                pass
-
-            def close():
-                pass
+    def test_upgrade_from_obsolete_version(self):
+        """We no longer support DB versions older than 17 (Tribler 6.0)"""
+        self.copy_and_initialize_upgrade_database('tribler_v12.sdb')
 
         db_migrator = DBUpgrader(self.session, self.sqlitedb, torrent_store=MockTorrentStore())
         self.assertRaises(VersionNoLongerSupportedError, db_migrator.start_migrate)
 
-    def test_upgrade_from_17(self):
-        pass
-        # TODO(emilon): Implement that one and 18 22 23
-        # assert sqlitedb.version == LATEST_DB_VERSION, "Database didn't get upgraded to latest version (%s != %s)" % (
-        #     sqlitedb.version, LATEST_DB_VERSION)
+    def test_upgrade_17_to_latest(self):
```

Comment:
@whirm I can place the file back. Do you think it's ok if I use the smaller database for the unit tests then? Maybe we can use the bigger databases for other unit tests in that case.